### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Andrey1104**

### CODE REVIEW

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]
Remove `db` from your PR and add just one migration. 
Adding just one migration per pull request (PR) is a recommended practice to maintain a clear and organized version history of your database schema changes in a Django project

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```
Comment (viktoria-rybenchuk) [inline .gitignore]: add a blank line in the end of the file

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]
You shouldn't delete migration that was before your changes `0001_initial.py`. You should just add one with your new changes

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
There are no changes done to the models that require migrations. Leave the migrations that are already present in the base repository in your code version so that migrations do not appear in the PR.

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**